index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    </head>
+    <body>
+        <form method="post">
+            <table id="table">
+                <tr>
+                    <th>Message</th>
+                    <th>Inferred label</th>
+                    <th colspan="2">Correctly labelled?</th>
+                </tr>
+            </table>
+            <input type="submit" value="Submit">
+        </form>
+        <script type="text/javascript" src="compiledJavaScript.js"></script>
```

Comment:
This PR doesn't add this - or the source needed to build it, so it won't really work after merging.